### PR TITLE
TorchFX: GPTQ accuracy fix

### PR DESCRIPTION
**Pull request by cavusmustafa**
### Details:
 - Fix for the accuracy issues discovered in Llama2 GPTQ with aot_autograd

### Tickets:
 - [CVS-149032](https://jira.devtools.intel.com/browse/CVS-149032)

### CODE REVIEW

**Review by rkazants — COMMENTED** [review]
Where is the test to check accuracy?

**Review by mvafin — APPROVED** [review]

**Comment by cavusmustafa** [comment]
> Where is the test to check accuracy?

Test for GPTQ will require to create a certain pattern and check if all the related passes are applied and the execution is optimized as expected. I am sharing the link to the ticket to track the progress on GPTQ tests and additional improvements. But I don't think it will be part of this PR and I don't think it will be ready in next 2-3 weeks due to other priorities.
https://jira.devtools.intel.com/browse/CVS-152181

**Comment by cavusmustafa** [comment]
[gptq_test.py.txt](https://github.com/user-attachments/files/17216763/gptq_test.py.txt)
@rkazants, could you please help deciding where to add this test? Is this a good location to implement it?: https://github.com/openvinotoolkit/openvino/tree/master/tests/model_hub_tests/pytorch
I need to skip model conversion and run it as a regular pytorch model after torch.compile. Can I do it in this structure?
Attached is the standalone script to test GPTQ patterns in a HF model.

**Comment by mvafin** [comment]
@cavusmustafa Add the test here https://github.com/openvinotoolkit/openvino/tree/master/tests/model_hub_tests/transformation_tests
You will not be bound to existing workflow and can create your custom test.

**Comment by cavusmustafa** [comment]
> @cavusmustafa Add the test here https://github.com/openvinotoolkit/openvino/tree/master/tests/model_hub_tests/transformation_tests You will not be bound to existing workflow and can create your custom test.

I just pushed into this directory. Can you please review?

**Review by mvafin — APPROVED** [review]

**Comment by mvafin** [comment]
@cavusmustafa  Please add the test in here:
https://github.com/openvinotoolkit/openvino/blob/500284d6bdcf979b4c4eda6abfa0b45cfe8f0635/.github/workflows/job_pytorch_models_tests.yml

You can refer to `PagedAttention Test` as example

**Comment by cavusmustafa** [comment]
> @cavusmustafa Please add the test in here: https://github.com/openvinotoolkit/openvino/blob/500284d6bdcf979b4c4eda6abfa0b45cfe8f0635/.github/workflows/job_pytorch_models_tests.yml
> 
> You can refer to `PagedAttention Test` as example

@mvafin, added the test into the ".yml" file. Please let me know if anything is missing. Thanks.

File: .github/workflows/job_pytorch_models_tests.yml [hunk 1]
```diff
@@ -149,6 +149,16 @@ jobs:
           TEST_DEVICE: CPU
           USE_SYSTEM_CACHE: False
 
+      - name: TorchFX GPTQ Pattern Test
+        timeout-minutes: 400
```
Comment (mvafin) [inline .github/workflows/job_pytorch_models_tests.yml:153]:
```suggestion
```

File: .github/workflows/job_pytorch_models_tests.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ permissions: read-all
 jobs:
   PyTorch_Models_Tests:
     name: PyTorch Models tests
-    timeout-minutes: ${{ inputs.model_scope == 'precommit' && 35 || 400 }}
+    timeout-minutes: ${{ inputs.model_scope == 'precommit' && 45 || 400 }}
```
Comment (mvafin) [inline .github/workflows/job_pytorch_models_tests.yml:25]:
```suggestion
    timeout-minutes: ${{ inputs.model_scope == 'precommit' && 120 || 400 }}
```

**Comment by mvafin** [comment]
Test fails

**Review by mvafin — APPROVED** [review]

**Pull request merged**